packages/elements/src/components/ino-avatar/ino-avatar.tsx

```diff
@@ -0,0 +1,95 @@
+import { Component, ComponentInterface, Prop, h, Element } from '@stencil/core';
+import classNames from 'classnames';
+import { hasSlotContent } from '../../util/component-utils';
+import primary from '../../assets/ino-avatar/primary.svg';
+import secondary from '../../assets/ino-avatar/secondary.svg';
+import primary_dashed from '../../assets/ino-avatar/primary_dashed.svg';
+import secondary_dashed from '../../assets/ino-avatar/secondary_dashed.svg';
+
+/**
+ * @slot icon-slot - For the icon to be displayed in the avatar.
+ */
+@Component({
+  tag: 'ino-avatar',
+  styleUrl: 'ino-avatar.scss',
+  shadow: false,
+})
+export class Avatar implements ComponentInterface {
+  @Element() el: HTMLElement;
+
+  /**
+   * The initials of the avatar.
+   */
+  @Prop() initials: string = '';
+
+  /**
+   * The img of the avatar.
+   */
+  @Prop() src: string = '';
+
+  /**
+   * Flag to enable hover effect.
+   */
+  @Prop() interactive: boolean = false;
```

Comment:
interactive mode should include keyboard accessibilty. As of this keyboard access with focus state should be added.